Configuration/Settings.yaml

```diff
@@ -34,6 +34,15 @@ Neos:
           'Neos.Neos.UI:HostOnlyStyles':
             resource: '${"resource://" + Neos.Ui.StaticResources.compiledResourcePackage() + "/Public/Styles/HostOnlyStyles.css"}'
             position: 'start 900'
+          'WorkSans:200':
```

Comment:
Also, if we switched from NotoSans to WorkSans (why?) you need to also change the family here: https://github.com/neos/neos-ui/blob/master/packages/neos-ui/src/styleHostOnly.css#L8